packs/clicrud/rules/gope_auto_remediate.yaml

```diff
@@ -0,0 +1,31 @@
+---
+description: 'GoPE remediate action for internet fail'
+tags: []
+type:
+  ref: standard
+  parameters:
+enabled: true
```

Comment:
Rules are usually somewhat installation specific so the convention is to ship them disabled by default - this way operators can configure them first and then enable them.